hveto/__main__.py

```diff
@@ -189,6 +189,9 @@ def create_parser():
               'this will launch automatically to condor, '
               'requires the gwdetchar package'),
     )
+    parser.add_argument('--no-submit', action='store_true',
+                        help='When omega scans are requested, do not submit DAG. Used when hveto is run in condor '
+                             'vanilla universe')
```

Comment:
I would recommend following the same style as the other arguments:

```suggestion
    parser.add_argument(
        '--no-submit',
        action='store_true',
        help=(
            'When omega scans are requested, do not submit DAG. '
            'Used when hveto is run in condor vanilla universe',
        )
```